docs/src/modules/reference/pages/cli/index.adoc

```diff
@@ -1,13 +1,307 @@
-= CLI
-:page-aliases: cli:index.adoc[]
+////
+index.adoc is auto-generated from
+- header template file "cli-index-header.template.txt"
+- all kalix*.adoc files
 
-include::ROOT:partial$include.adoc[]
+Please DON'T modify file index.adoc directly.
+Instead, you might want to modify file "cli-index-header.template.txt"
+////
+= CLI command reference
 
-Using the Akka CLI, you control all aspects of your Akka account from your command line. With it, you create and deploy new services, stream logs, and invite new developers to join your projects.
+The `akka` command line tool enables you to interact with your Akka projects and services using the command line. See xref:operations:cli/installation.adoc[] for information on how to download `akka`. Use xref:reference:cli/akka_version.adoc[`akka version`] to check the version.
 
-== Topics
+The following sections list xref:reference:cli/akka.adoc[akka] commands and their options.
 
-- xref:reference:cli/installation.adoc[]
-- xref:reference:cli/using-cli.adoc[]
-- xref:reference:cli/command-completion.adoc[]
-- xref:reference:cli/akka-cli/index.adoc[]
```

Comment:
instead `xref:reference:cli/akka-cli/index.adoc[]` now the content is here. Since we don't need this intermediate page no more.